Formula/o/opensca-cli.rb

```diff
@@ -0,0 +1,22 @@
+class OpenscaCli < Formula
+  desc "OpenSCA is a supply-chain security tool for security researchers and developers"
+  homepage "https://opensca.xmirror.cn"
+  url "https://github.com/XmirrorSecurity/OpenSCA-cli.git",
+      tag:      "v3.0.0",
+      revision: "f5e43fcd2fdbf2de9c5326f01f0395738dfec1b8"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -w -X 'main.version=#{version}'"
+    system "go", "build", *std_go_args(ldflags: ldflags),
+    "-o", "#{bin}/opensca-cli", "main.go"
```

Comment:
These are the default
```suggestion
    system "go", "build", *std_go_args(ldflags: ldflags)
```